### ISSUE TITLE
libretro-tyrquake (maybe others) cannot launch official Mission Packs.

### ISSUE THREAD

**Issue by sduensin** [RetroPie/RetroPie-Setup#2006]
After following the instructions on the wiki to install the official Quake Mission Packs, I discovered that libretro-tyrquake looks in the wrong place when it runs for the main Quake pak files.

For example, it looks for:
`/home/Pi/RetroPie/roms/quake/id1/hipnotic/pak?.pak`
and:
`/home/Pi/RetroPie/roms/quake/id1/id1/pak?.pak`

Obviously `id1/id1` is invalid.  You can fix the issue with an ugly symlink to create an `id1/id1` or you can move the `hipnotic` and `rogue` folders to be parallel to `id1` and update the shell scripts that start the games.

**Comment by joolswills** [comment]
What version of RetroPie-Setup script ?

**Comment by psyke83** [comment]
@joolswills,

@sduensin is partially correct. It's true that the folder structure is incorrect, but this is the proper layout:

ports/quake/id1
ports/quake/hipnotic
ports/quake/rogue

i.e. the mission pack folders should be parallel and not subfolders of id1. I've sent a PR: https://github.com/RetroPie/RetroPie-Setup/pull/2054

The Wiki will also need to be updated to reflect this change.

FYI (slightly off-topic) - I purchased the Quake pack on Steam in order to test this, but none of the Quake entries launched successfully for me after installing lr-tyrquake (using latest 4.2.12 script) and full version paks, unrelated to the problem with file paths. I tried installing from binary and built from source, but no difference. The runcommand.log always showed "Illegal instruction".

I manually built against this commit (which would have been the latest from when this issue was opened) and now it launches properly: https://github.com/libretro/tyrquake/commit/12d7ca3b04a4d445edd3ef03d303bca6de466134

It's an upstream issue so I'll bisect and try to solve the problem when I have time, but just letting you know in case it causes further confusion.
